lib/features/home_screen/widgets/home_screen_app_bar.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app_kids/core/app/pages.dart';
+import 'package:givt_app_kids/features/flows/cubit/flows_cubit.dart';
+import 'package:givt_app_kids/features/home_screen/cubit/navigation_cubit.dart';
+import 'package:givt_app_kids/features/profiles/cubit/profiles_cubit.dart';
+import 'package:givt_app_kids/helpers/analytics_helper.dart';
+import 'package:go_router/go_router.dart';
+
+class HomeScreenAppBar extends StatelessWidget implements PreferredSizeWidget {
+  const HomeScreenAppBar({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final profiles = context.watch<ProfilesCubit>();
+    final navigation = context.watch<NavigationCubit>();
+    return AppBar(
+      title: Text(
+        navigation.state.activeDestination.appBarTitle.isEmpty
+            ? profiles.state.activeProfile.firstName
+            : navigation.state.activeDestination.appBarTitle,
+        style: Theme.of(context).textTheme.titleLarge,
+      ),
+      backgroundColor: Theme.of(context).colorScheme.onPrimary,
+      systemOverlayStyle: SystemUiOverlayStyle(
+        statusBarColor: Theme.of(context).colorScheme.onPrimary,
+        statusBarIconBrightness: Brightness.dark, // For Android (dark icons)
+        statusBarBrightness: Brightness.light, // For iOS (dark icons)
+      ),
+      actions: [
+        if (navigation.state.activeDestination.name == 'home')
```

Comment:
why not just (navigation.state.activeDestination == NavigationDestinationData.home)?